eng/tox/tox.ini

```diff
@@ -326,8 +326,9 @@ changedir = {envtmpdir}
 setenv =
   {[testenv]setenv}
   PROXY_URL=http://localhost:5014
+  PIP_EXTRA_INDEX_URL=
 deps =
-  {[base]deps}
+  -rdev_requirements.txt
```

Comment:
dev_requirements, no longer including `test_tools.txt`.

The install of apiview_reqs.txt is where stub generator comes in.